src/main/java/com/pubnub/kafka/connect/PubNubKafkaSinkConnectorTask.java

```diff
@@ -18,6 +20,16 @@
 public class PubNubKafkaSinkConnectorTask extends SinkTask {
     private final Logger log = LoggerFactory.getLogger(this.toString());
 
+    @VisibleForTesting
```

Comment:
I am assuming that it is difficult to add PubNub instance to constructor of the class.

Reply:
changed to constructor